lib/sqlalchemy/sql/selectable.py

```diff
@@ -6724,7 +6765,7 @@ def select_from(self: SelfExists, *froms: FromClause) -> SelfExists:
 
         """
         e = self._clone()
```

Comment:
**mike bayer** (zzzeek) wrote:

remove

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325